src/05-glue/04-optparse.md

```diff
@@ -227,7 +227,7 @@ fmap id = id
 fmap (f . g) == fmap f . fmap g
 ```
 
-Any type `f` that can implement `fmap` and follow these laws can be an
+Any type `f` that implements `fmap` and follows these laws is an
```

Comment:
```suggestion
Any type `f` that can implement `fmap` and follow these laws can be a valid
```

I think the old phrasing is important, as it refers not existing types, but future types (perhaps written by the learner) as well.